cli/cmd/component.go

```diff
@@ -150,70 +193,116 @@ func componentsToTable() [][]string {
 		out = append(out, []string{
 			cdata.Status().String(),
 			cdata.Name,
+			cdata.LatestVersion.String(),
 			cdata.Description,
 		})
 	}
 	return out
 }
 
-func runComponentsInstall(_ *cobra.Command, args []string) error {
-	cacheDir, err := cache.CacheDir()
+func runComponentsInstall(_ *cobra.Command, args []string) (err error) {
```

Comment:
I was thinking that `ComponentType` would be an interface and each type would implement install/update/etc. methods, but this might be fine for now since we only have a single component.